### ISSUE TITLE
Some content is not usable when you enlarge the text

### ISSUE THREAD

**Issue by nickcolley** [alphagov/tech-docs-gem#127]
Some elements have fixed font sizes, for example `<code>` elements.

<img width="574" alt="Documentation example with font scaling inconsistently" src="https://user-images.githubusercontent.com/2445413/66841294-6234bb80-ef61-11e9-813d-12d7fc631acf.png">

https://tdt-documentation.london.cloudapps.digital/multipage.html#build-a-multipage-site

These elements are sized using pixels which is why they do not scale.

## Failing WCAG Success Criteria
- [1.4.4: Resize text - Level AA](https://www.w3.org/WAI/WCAG21/quickref/#resize-text)

**Comment by selfthinker** [comment]
I know many auditors who wouldn't interpret this as a WCAG fail, mostly because zooming didn't exist when WCAG was written. See also https://alastairc.ac/2017/11/is-text-sizing-dead/
I'm on the fence about this one and would generally love to see more research on how people make text bigger. I would consider a "loss of content or functionality" a definite fail, that text size doesn't increase not necessarily, depending on how bad it is.

**Comment by nickcolley** [comment]
I think if you required a very large text size you would not be able to read these elements.

It also seems to be a fail based on the wording of the criteria:

"Except for captions and images of text, text can be resized without assistive technology up to 200 percent without loss of content or functionality."

**Comment by bravegrape** [comment]
This is very interesting! @nickcolley did you get this result with the zoom tool in the Empathy Lab? It's curious that keyboard zooming in the browser (I tried Chrome and Safari) doesn't lead to this inconsistency 🤔

**Comment by nickcolley** [comment]
There's two ways to do zooming, the one you're familiar with and text zooming. Text zooming requires you to update settings in your browser.

**Comment by nickcolley** [comment]
On reflection I agree with Anika, we should fix this but I don't think it's a WCAG failure since the content is not obscured.

**Comment by selfthinker** [comment]
Just to clarify, there are lots more ways to increase text. Apart from browser zoom and browser text sizing, there are also different kinds of magnification, OS text increase, custom styles, changing the display resolution, using physical magnifiers, etc.
(Sorry for being nitpicky, this is not really related to the issue, just wanted to clarify a potential misconception.)
